Dockerfile

```
@@ -1,34 +1,38 @@
-FROM kbase/kbase:sdkbase.latest
+FROM kbase/sdkpython:3.8.10
 MAINTAINER KBase Developer
 # -----------------------------------------
-
-# Insert apt-get instructions here to install
-# any required dependencies for your module.
-
-# RUN apt-get update
-
-# update java
-RUN add-apt-repository ppa:openjdk-r/ppa \
-	&& sudo apt-get update \
-	&& sudo apt-get -y install openjdk-8-jdk \
+# Install any required dependencies for your module
+RUN apt-get update \
+	&& apt-get install -y wget unzip openjdk-8-jdk \
 	&& echo java versions: \
 	&& java -version \
 	&& javac -version \
 	&& echo $JAVA_HOME \
 	&& ls -l /usr/lib/jvm \
-	&& cd /kb/runtime \
-	&& rm java \
 	&& ln -s /usr/lib/jvm/java-8-openjdk-amd64 java \
-	&& ls -l
+	&& ls -l \
+	&& rm -rf /var/lib/apt/lists/*
 
 ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
 
-# update jars
-RUN cd /kb/dev_container/modules/jars \
-	&& git pull \
-	&& . /kb/dev_container/user-env.sh \
-	&& make deploy
+RUN pip install configparser
+
+# Clone KBase jars repository
+RUN git clone https://github.com/kbase/jars /tmp/repo
+RUN cp -r /tmp/repo/* /kb/deployment
```

```suggestion
RUN mv -r /tmp/repo/* /kb/deployment && rm -r /tmp/repo
```
otherwise you have 2 copies of the jars repo, which is big.

---

👍

---

:+1: